diff --git a/ui/src/core/embedder/default_plugins.ts b/ui/src/core/embedder/default_plugins.ts
index 1ba9073306..a57bb51442 100644
--- a/ui/src/core/embedder/default_plugins.ts
+++ b/ui/src/core/embedder/default_plugins.ts
@@ -44,6 +44,7 @@ export const defaultPlugins = [
   'com.android.PinSysUITracks',
   'com.android.SysUIWorkspace',
   'com.android.WearLongBatteryTracing',
+  'com.google.PerfettoMcp',
   'com.google.PixelCpmTrace',
   'com.google.PixelMemory',
   'dev.perfetto.AggregateProfiles',
